runtime/runtime-core/common/test/aws/smithy/kotlin/runtime/retries.impl/ExponentialBackoffWithJitterTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package aws.smithy.kotlin.runtime.retries.impl
```

Comment:
fix

The file location of this looks to be `runtime/retries.impl` rather than `runtime/retries/impl`